lib/ex_unit/lib/ex_unit/server.ex

```diff
@@ -68,6 +77,22 @@ defmodule ExUnit.Server do
       %{config | captured_devices: Set.delete(config.captured_devices, device)}}
   end
 
+  def handle_call({:log_capture_on, pid}, _from, config) do
+    ref = Process.monitor(pid)
+    case config.log_captures do
```

Comment:
Good point, I've pushed the changes for capture handler monitoring: https://github.com/lexmag/elixir/commit/09f21d9e823b23355a9abbb63bac29c967929a66.
How do you think we can notify `capture_log` about `_reason` ([here](https://github.com/lexmag/elixir/commit/09f21d9e823b23355a9abbb63bac29c967929a66#diff-d7e207db0244b0fb34ee0f2e04d83e2eR142)) on `:gen_event_EXIT` message?